src/test/java/bio/terra/workspace/service/datareference/DataReferenceServiceTest.java

```diff
@@ -89,6 +131,23 @@ public void testGetDataReferenceByName() {
     assertThat(ref.name(), equalTo(request.name()));
   }
 
+  @Test
+  public void testGetDataReferenceByWrongTypeMissing() {
+    UUID workspaceId = createDefaultWorkspace();
+    DataReferenceRequest request =
+        defaultReferenceRequest(workspaceId, DataReferenceType.DATA_REPO_SNAPSHOT).build();
+    String snapshotName = request.name();
+    dataReferenceService.createDataReference(request, USER_REQUEST).referenceId();
```

Comment:
why aren't we doing anything with the reference id?

Reply:
whoops, it was previously an unused variable and I removed the var but not `.referenceId()`. We should be calling `createDataReference()` and ignoring the return value.

Reply:
glad we're ignoring the right value 😥